src/atoms/forms/submit-button.test.tsx

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import { render } from "@testing-library/react";
+import { SubmitButton } from "./submit-button";
+import faker from "faker";
+import { ButtonStyles } from "../constants/button-styles";
+
+describe("SubmitButton", () => {
+    test("when default props, renders default buttonText", async () => {
+        // Arrange
+        const expected = "Submit";
+
+        // Act
+        const { getByText } = render(<SubmitButton />);
+
+        // Assert
+        expect(getByText(expected)).not.toBeNull();
+    });
+    test("when buttonText provided, renders supplied buttonText", async () => {
+        // Arrange
+        const expected = faker.random.words();
+
+        // Act
+        const { getByText } = render(<SubmitButton buttonText={expected} />);
+
+        // Assert
+        expect(getByText(expected)).not.toBeNull();
+    });
+
+    test("when cssClassName provided, renders with className set", async () => {
```

Comment:
Question - do these tests need to be `async`? I assume they're passing right now, so it's no harm - but could be confusing for a reader

Reply:
Totally a copy paste oversight